pkgs/applications/networking/mailreaders/dodo/default.nix

```diff
@@ -0,0 +1,31 @@
+{ pkgs, lib, python3 }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "dodo";
+  version = "2022-08-13";
+  format = "pyproject";
+
+  src = pkgs.fetchFromGitHub {
+    repo = pname;
+    owner = "akissinger";
+    rev = "b377e05ab507c7bec427e3bac0c35b68bd67d9fc";
+    sha256 = "sha256-GcOcCB93/q84zPdiY8CgZN9MQtG2BG0miGsasonmofs=";
+  };
+
+  nativeBuildInputs = with python3.pkgs; [ setuptools-scm ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    bleach
+    pyqt5
+    pyqt5_sip
+    pyqtwebengine
+  ];
+
+  meta = with lib; {
+    description = "A graphical, hackable email client based on notmuch ";
+    homepage = "https://github.com/akissinger/dodo";
+    license = licenses.gpl3;
```

Comment:
Please consult [this](https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206) and change the license attribute accordingly.